lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.scss

```diff
@@ -2,16 +2,26 @@
     &-metadata-properties {
         mat-expansion-panel-header {
             height: 56px;
+            padding: 0 12px;
 
             .adf-metadata-properties-title {
-                font-weight: normal;
-                font-size: 15px;
+                font-weight: 700;
+                font-size: 14px;
                 padding-left: 12px;
             }
         }
 
         .mat-expansion-panel:not([class*='mat-elevation-z']) {
             box-shadow: none;
+            border: 1px solid var(--adf-metadata-property-panel-border-color);
```

Comment:
I think you have to change the variable name here. am I right?

Reply:
Updated